features/fixtures/TestApp/config/config.yml

```diff
@@ -36,7 +39,26 @@ dunglas_api:
                 default:               3
                 client_can_change:     true
 
```

Comment:
Huh? I 'll try it but seems weird for two reasons:
• isn't this parameter supposed to be to true by default if the
FOSUserBundle is enalbled?
• I took the configuration of my project and I do not have it...

On Saturday, August 8, 2015, Maxime Steinhausser <notifications@github.com>
wrote:

> In features/fixtures/TestApp/config/config.yml
> <https://github.com/dunglas/DunglasApiBundle/pull/211#discussion_r36577656>
> :
>
> > @@ -36,7 +39,26 @@ dunglas_api:
> >                  default:               3
> >                  client_can_change:     true
> >
>
> missing enable_fos_user: true in the dunglas_api configuration.
> That's why your listener is never called I guess.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dunglas/DunglasApiBundle/pull/211/files#r36577656>.
>